os/mm/mm_heap/mm_getsize.c

```diff
@@ -90,3 +90,37 @@ size_t mm_get_largest_freenode_size(void)
 
 	return largest_size;
 }
+
+/****************************************************************************
+ * Name: mm_get_heap_free_size
+ *
+ * Description:
+ *   returns free size in the heap
+ ****************************************************************************/
+#ifdef CONFIG_DEBUG_MM_HEAPINFO
+size_t mm_get_heap_free_size(void)
```

Comment:
I think we can make these as inline functions.

Reply:
I will do refactoring most of the functions in `mm_getheap.c` and `mm_getsize.c` about separating umm and kmm.
I will change it at that time.
Thanks:)